### PR TITLE
updated html for event_home and event_home_owned

### PR DESCRIPTION
**Pull request by LoganTown**
Adjusted / reworked html for event_home and event_home_owned

doesnt have navbar, just included button to go to dash

### CODE REVIEW

**Review by benyeom — DISMISSED** [review]
maybe not have bullet points? and change the color scheme

**Comment by gyishak** [comment]
> maybe not have bullet points? and change the color scheme

Do you mean for the available roles listed?

**Comment by LoganTown** [comment]
Why no bullet points? its a list

**Comment by gyishak** [comment]
> Why no bullet points? its a list

I agree, I like the list too, I also need the list for the signing up for roles feature.

**Review by gyishak — DISMISSED** [review]

**Review by mnugent04 — COMMENTED** [review]
made a commit just for colors and made the attendees, invitees, and roles into boxes like wire frame.

**Comment by mnugent04** [comment]
other than that looks good, did we want to extend the base html to this page?

**Review by mnugent04 — APPROVED** [review]

File: our_site/planner/templates/planner/event_home_owned.html [hunk 1]
```diff
@@ -1,40 +1,91 @@
...
+                </div>
+                <div class="column has-text-left box">
+                    <h2 class="subtitle is-4">Invitees:</h2>
+                    <ul>
+                        {% for invitee in invitees %}
+                            <li>{{ invitee.user.username }}</li>
+                        {% endfor %}
+                    </ul>
+                </div>
+                <div class="column has-text-left box">
```
Comment (Curtis-Barnhart) [inline our_site/planner/templates/planner/event_home_owned.html:67]: third box is always larger than other two?? are we okay removing this cause it looks pretty jank with it. Or could we possibly find a way to fix it? I tried looking a little but couldn't figure out what was causing it.

File: our_site/planner/templates/planner/event_home.html [hunk 2]
```diff
@@ -1,76 +1,84 @@
...
+                    <h2 class="subtitle is-4">Invitees:</h2>
                     <ul>
-                        {% for attendee in attendees %}
-                            <li>{{ attendee.user.username }}</li>
+                        {% for invitee in invitees %}
+                            <li>{{ invitee.user.username }}</li>
                         {% endfor %}
                     </ul>
                 </div>
+                <div class="column has-text-left box">
```
Comment (Curtis-Barnhart) [inline our_site/planner/templates/planner/event_home.html:57]: See other comment, same problem here.

**Review by Curtis-Barnhart — CHANGES_REQUESTED** [review]

**Review by Curtis-Barnhart — APPROVED** [review]
I am okay without boxes

**Review by gyishak — CHANGES_REQUESTED** [review]
Why did we take out the bullet points?

**Review by mnugent04 — APPROVED** [review]

**Review by gyishak — APPROVED** [review]

**Pull request merged**